.gitignore

```diff
@@ -127,3 +127,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# IDE
+.idea/
```

Comment:
Why was this not yet part of the .gitignore ?

Reply:
Clear, thanks, some documentation tackling this exact topic : https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/